person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age
+    @age >= 18
+  end
```

Comment:
- [ ] As per the project [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md),  `Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.`, please kindly make the method of_age?  private. Something like so: 

```ruby
  private 
  def of_age?
    @age >= 18
  end
```

Reply:
Thank you @Mihndim2020 . This has been taken care of.